### PR TITLE
[#9205] "Resend" option doesn't resend message on Android

### LINKED ISSUE #9205: "Resend" option doesn't resend message on Android
# Bug Report

## Problem
"Resend" option doesn't work on android devices (tried on LG V20, Android 8).
Also popup is not closed when tapping on the blurred area.

#### Expected behavior
can resend message

#### Actual behavior
photo_2019-10-14 16 27 18 jpeg 2019-10-14 16-27-31

### Reproduction
 **A bit complicated to emulate this behavior if you have strong connection and fast device.
Below are steps that working for me.**

- Open Status
- Join public chat
- Go offline 
- Go online and quickly until "Connecting" is shown send message
- Turn off WI-FI
- Wait ~1 min
- Turn on WI-FI and after a while you can see "Not confirmed. Tap for options."
- Tap on it, try to resend message

### Additional Information

- Status version: nightly 14/10/2019
- Operating System: Android

#### Logs

[comment]: # (if on Android please replicate bug whilst running adb logcat)
Full logcat: 
[resend_button.log](https://github.com/status-im/status-react/files/3724862/resend_button.log)

```
10-14 16:20:34.652 30863 30901 D ReactNativeJS: DEBUG [status-im.utils.handlers:32] - Handling re-frame event:  :signals/signal-received
10-14 16:20:34.661 30863 30901 D ReactNativeJS: DEBUG [status-im.events:667] - :event-str {"type":"wallet","event":{"type":"newblock","blockNumber":8739634,"accounts":[]}}
10-14 16:20:41.407 30863 30901 D ReactNativeJS: DEBUG [status-im.utils.handlers:32] - Handling re-frame event:  :chat.ui/set-chat-ui-props
10-14 16:20:42.167 30863 30901 D ReactNativeJS: DEBUG [status-im.utils.handlers:32] - Handling re-frame event:  :chat.ui/set-chat-ui-props
10-14 16:20:45.605 30863 30901 D ReactNativeJS: DEBUG [status-im.utils.handlers:32] - Handling re-frame event:  :signals/signal-received
10-14 16:20:45.615 30863 30901 D ReactNativeJS: DEBUG [status-im.events:667] - :event-str {"type":"wallet","event":{"type":"newblock","blockNumber":8739635,"accounts":[]}}
10-14 16:20:54.351 30863 30901 D ReactNativeJS: DEBUG [status-im.utils.handlers:32] - Handling re-frame event:  :signals/signal-received
10-14 16:20:54.361 30863 30901 D ReactNativeJS: DEBUG [status-im.events:667] - :event-str {"type":"wallet","event":{"type":"newblock","blockNumber":8739636,"accounts":[]}}
```

### PR DESCRIPTION
**Pull request by tbenr**
fixes #9205 

this is a sort of workaround since I changed options to a `sheet` UI style. This seems to me more coherent with current sheet usage around the app. A good side-effect is that this UI seems not be effected by the strange behaviour.

image

ping @rachelhamlin 

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/chat/message/message.cljs [hunk 1]
```diff
@@ -146,24 +147,14 @@
   [chat-id message-id]
   [react/touchable-highlight
    {:on-press (fn [] (cond
```
Comment (rasom) [inline src/status_im/ui/screens/chat/message/message.cljs]: it could be just `if` instead of `cond` now

**Review by rasom — APPROVED** [review]

**Comment by churik** [comment]
e2e: known failures, not related to PR.

**Comment by churik** [comment]
@tbenr good job, thank you!
however, one issue should be fixed.
In most of the cases on chat view the keyboard is opened, and in this case if you tap on "Not confirmed. Tap for options.", keyboard overlaps menu with options.


photo_2019-10-28 09 27 20

All options are working fine on Android and IOS.

**Comment by tbenr** [comment]
@churik oh, right. Let me fix it.

**Comment by tbenr** [comment]
@churik fixed!

**Comment by churik** [comment]
All options are checked and they are working on IOS 11.4.1 and Android 8.0

Awesome work and thanks for quick fix @tbenr !

**Pull request merged**